libs/ui/elements/src/lib/markdown-parser/markdown-parser.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing'
+import { MarkdownParserComponent } from './markdown-parser.component'
+
+describe('MarkdownParserComponent', () => {
+  let component: MarkdownParserComponent
+  let fixture: ComponentFixture<MarkdownParserComponent>
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [MarkdownParserComponent],
+    }).compileComponents()
+
+    fixture = TestBed.createComponent(MarkdownParserComponent)
+    component = fixture.componentInstance
+    fixture.detectChanges()
+  })
+
+  it('should create', () => {
+    expect(component).toBeTruthy()
+  })
```

Comment:
Oh sorry, I didn't realize that in the first review but we we need at least one test here, showing that a markdown input will be correctly transformed to HTML; this doesn't have to be exhaustive as we're not testing the underlying library, but simply making sure that the test is processed.

Could you please add that?

Reply:
Oh, i totally forgot about that.

I created 3 tests described like this :
- should render markdown as HTML
- should render HTML as HTML
- should render text as HTML 

Is it too much ? Maybe the first test is enough.

Reply:
That does look like a perfect amount of tests :slightly_smiling_face: